lib/fizzy/execute.cpp

```diff
@@ -131,7 +131,7 @@ inline uint64_t popcnt64(uint64_t value) noexcept
 }
 }  // namespace
 
-Instance instantiate(const Module& module)
+Instance instantiate(const Module& module, std::vector<ImportedFunction> imported_functions)
```

Comment:
I think at some point we might introduce a struct to pass all imports as one parameter, like
```
struct Imports
{
  std::vector<ImportedFunction> imported_functions
  std::vector<uint64_t> globals;
  std::optional<bytes> memory; // not sure yet if it's data bytes or just limits that can be imported 
  // table
};
```
In this PR I'd leave it as only vector of functions.